watchtower/wtclient/client.go

```diff
@@ -1319,16 +1319,17 @@ func (c *TowerClient) handleStaleTower(msg *staleTowerMsg) error {
 
 // RegisteredTowers retrieves the list of watchtowers registered with the
 // client.
-func (c *TowerClient) RegisteredTowers(opts ...wtdb.ClientSessionListOption) (
-	[]*RegisteredTower, error) {
+func (c *TowerClient) RegisteredTowers(includeExhaustedSessions bool,
```

Comment:
nit: it may be confusing that here the bool indicates inclusion, while on the RPC level we have `ExcludeExhaustedSessions` indicating the opposite.

Reply:
What do we need to do

Reply:
Perhaps make them indicate the same, e.g by making the RPC parameter into `IncludeExhaustedSessions`

Although after thinking about it for a bit, it depends on what we want the default behavior to be. If the default should be to include **all** sessions, then it's more "correct" to have the user define exclusion, when desired.

If default should be to exclude exhausted sessions, then the parameter should indicate the inclusion.

Reply:
yeah my main reasoning here is: I would like the default to one day be to exclude exhausted sessions but cant do this from the get go as it would be a breaking change for anyone using this API rn.

Reply:
happy to change everything to `Exclude` if reviewers would prefer!

Reply:
went another way 🤓  so gonna close this part